src/morse/index.js

```diff
@@ -58,6 +58,42 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  let temp = '';
+   if (Array.isArray(text) || text == undefined) {
```

Comment:
Why not just check the type? You want to make sure that what you're given is a string, not that its an iterable. An array of numbers for example would pass this check but isn't necessarily useful to you.